database/seed.spec.js

```diff
@@ -1,7 +1,5 @@
 const mongoose = require('mongoose');
-const seed = require('./seed.js');
 const dbCollections = require('./RoomAndReview.js');
-const util = require('./util.js');
 
 describe('db manipulation', () => {
   let connection;
```

Comment:
It looks like this is only testing the mongoose driver's functions, which isn't necessary. Do the seed or dbCollections modules have their own functions or code that need to be tested?